docs/DeveloperGuide.md

```diff
@@ -334,8 +334,9 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
 2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-
-*{More to be added}*
+4.  Should be usable by a novice who has never used a command line interface.
+5.  Common actions should require few input actions to accomplish.
```

Comment:
the `few input actions` sounds a little ambiguous. 
Perhaps can consider quantifying it like `less than x number of phrases`?